src/prpy/tsr/tsr.py

```diff
@@ -175,8 +199,8 @@ def rpy_within_bounds(rpy, Bw):
         @param Bw bounds on rpy
         @return check a (3,) vector of True if within and False if outside
         """
-        # Unwrap rpy to [-pi, pi].
-        rpy = (numpy.array(rpy) + pi) % (2*pi) - pi
+        # Unwrap rpy to Bw_cont.
+        rpy = wrap_to_interval(rpy, lower=Bw[:, 0])
```

Comment:
I thought the purpose of creating `Bw_cont` was to unify the treatment of all angular ranges. However, in this code, we ignore `Bw_cont` and implement separate logic for inner and outer intervals. Is this intentional?